### DISCUSSION TITLE
Very large public folder size

### DISCUSSION THREAD

**Discussion by igel2000** [thegeeklab/hugo-geekdoc#335] [Q&A] [answered]
Example project - https://github.com/igel2000/test_hugo_geekdocs.
Content folder size - 250Kb. Public folder size - 1.6Gb.
One of my real project has 10Gb in public folder.
Сould you help me figure it out how  to reduce the size of the public folder?

**Comment by xoxys** [comment]
You have a very huge and deep page structure. This results in a complex HTML structure for the sidebar navigation, which is fully included in every single rendered HTML page. The size of the `public/` directory can be reduced by minifying the HTML **but** there is a downside described in the theme's documentation https://geekdocs.de/usage/getting-started/#minify-html-results-in-spacing-issues.

Excluding HTML files at all will not help in your case as these files are the root cause, but you can try a bit more restrictive configuration (need to be added to your project `config.toml`):

```
[minify]
[minify.tdewolff]
[minify.tdewolff.html]
keepDocumentTags = true
keepEndTags = true
keepQuotes = true
keepWhitespace = true
```

**Comment by igel2000** [comment, marked as answer]
I tried `hugo --minify` earlier. But it didn't work for me.
After update contig.toml have 4Gb instead 10Gb. It isn't perfect yet, but very better than before.
Thank you for your idea.

**Reply by xoxys** [reply]
Don't really know how to improve it further, sorry. If you find something we would love to hear about it :)
